src/components/EditableHeading/__tests__/EditableHeading-interactions.tsx

```diff
@@ -0,0 +1,73 @@
+import { expect } from "@storybook/jest";
+import {
+  Canvas,
+  clearText,
+  clickElement,
+  getByTestId,
+  getByRole,
+  interactionSuite,
+  typeText
+} from "../../../tests/interactions-utils";
+
+import { ComponentDefaultTestId } from "../../../tests/constants";
+import { resetFocus } from "../../../__tests__/interactions-helper";
+
+const text = "This heading is an editable heading";
+
+function getComponent(canvas: Canvas) {
+  return getByTestId(canvas, ComponentDefaultTestId.CLICKABLE);
+}
+function getHeading(canvas: Canvas) {
+  return getByTestId(canvas, ComponentDefaultTestId.CLICKABLE);
+}
+
+function getInput(canvas: Canvas) {
+  return getByRole(canvas, "input");
+}
+
+async function testBasicEditing(canvas: Canvas) {
```

Comment:
I try to name the tests by what they should do.
For example, `editAndChangeToValidText`

That forces me to write tests that are simple and doesn't mix multiple tests in one